gha.Makefile

```diff
@@ -0,0 +1,127 @@
+ifdef DOCKER_USER
+	DOCKER_IMAGE_NAME ?= ${DOCKER_USER}/actions-runner-controller
+else
+	DOCKER_IMAGE_NAME ?= ghcr.io/actions-runner-controller
+endif
+
+DOCKER_USER ?= $(shell echo ${DOCKER_IMAGE_NAME} | cut -d / -f1)
+
+YQ=$(shell )
+
+IMAGE_VERSION ?= dev
+BINARY_VERSION ?= $(shell cat ./charts/gha-runner-scale-set-controller/Chart.yaml | yq .version)
+
+# default list of platforms for which multiarch image is built
+ifeq (${PLATFORMS}, )
+	export PLATFORMS="linux/amd64,linux/arm64"
+endif
+
+# if IMG_RESULT is unspecified, by default the image will be pushed to registry
+ifeq (${IMG_RESULT}, load)
+	export PUSH_ARG="--load"
+	# if load is specified, image will be built only for the build machine architecture.
+	export PLATFORMS="local"
+else ifeq (${IMG_RESULT}, cache)
+	# if cache is specified, image will only be available in the build cache, it won't be pushed or loaded
+	# therefore no PUSH_ARG will be specified
+else
+	export PUSH_ARG="--push"
+endif
```

Comment:
It's not very clear how this can be used 🤔 we either need to document the whole flow or use another approach